### ISSUE TITLE
Suggestion:

### ISSUE THREAD

**Issue by DrDarkfire** [Leviathan143/betterbeginnings-MODIFIED#55]
Tough as Nails Support Suggestion:
Minecraft: 1.10.2

I think it would be beneficial that the campfire in this mod acts like the one in tough as nails and gives off heat so if someone like me who is working on a pack can have just one campfire that has the functionality of both mods when they are both installed. Please and thank you.

**Comment by Leviathan143** [comment]
Unfortunately this is not possible in released versions of TAN for 1.10.2. It is supported in unreleased versions, and in 1.11.2 TAN. I've marked this as todo-1.11.2 and I'll be leaving it open to remind me to look into this again when the time comes to do a 1.11.2 port of BB.

**Comment by hron84** [comment]
Just question: what about other side? TAN campfire could be used in this mod in any way?

**Comment by Leviathan143** [comment]
@hron84 No, not even remotely possible. The TAN campfire is purely a heat source, it is incapable of processing items. In any case, the hooks I need are available in later versions of TAN.

**Comment by hron84** [comment]
@Leviathan143 ohh, i see.
